webapp/channels/src/utils/paste.test.tsx

```diff
@@ -170,6 +170,33 @@ describe('formatMarkdownLinkMessage', () => {
         const formatttedMarkdownLinkMessage = formatMarkdownLinkMessage({selectionStart: 0, selectionEnd: 4, message, clipboardData});
         expect(formatttedMarkdownLinkMessage).toEqual('[test](https://example.com/)');
     });
+
+    test('Should not add link when pasting inside of a formatted markdown link', () => {
+        const message = '[test](url)';
+        const formatttedMarkdownLinkMessage = formatMarkdownLinkMessage({selectionStart: 7, selectionEnd: 10, message, clipboardData});
+        expect(formatttedMarkdownLinkMessage).toEqual('https://example.com/');
```

Comment:
Shouldn't this case result in: `[test](https://example.com/)` as only `url` is selected?

Reply:
The functions doesnt return the complete text but the resultant text so this is expected output.

Reply:
Ah, ok - it was confusing because the other case that is not selected `url` did return the full markdown text